packages/react-native/React/Base/RCTConvert.m

```diff
@@ -1487,6 +1487,105 @@ + (NSPropertyList)NSPropertyList:(id)json
     integerValue)
 
 #if TARGET_OS_OSX // [macOS
+// Used to convert the JSON from `role` to the corresponding
+// NSAccessibilityRole (NSString*), has slightly different mappings than the
+// old accessibilityRole prop for back compatability. `role` matches ARIA Core
+// AAM spec and takes precedence. See https://www.w3.org/TR/core-aam-1.1/
++ (NSString*) accessibilityRoleFromRole:(NSString*)ariaRole
```

Comment:
The argument is an `NSString *` here but an `id` in the header. I didn't even know this was legal in Objective C.